TEST patch for support dm7080.
Based on:
https://github.com/openatv/enigma2/commit/20f8b23707eeef4fcd2ab284b7f0fe95461a7dfb#diff-67e997bcfdac55191033d57a16d1408a

Tryfix for 'Systemplugins/Videomode([Errno 2] No such file or directory: '/proc/stb/denc/0/wss')' problem. 

diff --git a/lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py b/lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py
index 74ba0f1..3ee3ecd 100644
--- a/lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py
+++ b/lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py
@@ -357,10 +357,14 @@ class VideoHardware:
 		else:
 			wss = "auto"
 
-		print "[VideoHardware] -> setting aspect, policy, policy2, wss", aspect, policy, policy2, wss
+		print "[VideoHardware] -> setting aspect, policy, policy2", aspect, policy, policy2
 		open("/proc/stb/video/aspect", "w").write(aspect)
 		open("/proc/stb/video/policy", "w").write(policy)
-		open("/proc/stb/denc/0/wss", "w").write(wss)
+		if os.path.exists("/proc/stb/denc/0/wss"):
+			print "[VideoMode] setting wss: %s" % wss
+			f = open("/proc/stb/denc/0/wss", "w")
+			f.write(wss)
+			f.close()
 		try:
 			open("/proc/stb/video/policy2", "w").write(policy2)
 		except IOError:
diff --git a/lib/python/Tools/StbHardware.py b/lib/python/Tools/StbHardware.py
index 2da266c..92b84d8 100644
--- a/lib/python/Tools/StbHardware.py
+++ b/lib/python/Tools/StbHardware.py
@@ -6,7 +6,7 @@ from time import time, localtime, gmtime
 def getFPVersion():
 	ret = None
 	try:
-		ret = long(open("/proc/stb/fp/version", "r").read())
+		ret = open("/proc/stb/fp/version", "r").read()
 	except IOError:
 		try:
 			fp = open("/dev/dbox/fp0")
